sdk/burdock/query/sql/private/query.py

```diff
@@ -52,17 +53,40 @@ def _apply_noise(self, subquery, query, syms, types, sens, srs):
             name, sym = nsym
             name = name.lower()
             sens = sym.sensitivity()
+            
             if sym.type() == "int":
                 if sym.sensitivity() == 1:
-                    counts = self.mechanism.count(srs[name])
-                    counts[counts < 0] = 0
-                    srs[name] = counts
-                    srs = srs.filter(name, ">", self.tau)
+                    if(bounds == "lower"):
+                        lower_bound = srs[name] + self.mechanism.bounds(pct)[0]
+                        srs[name] = lower_bound
+                    elif(bounds == "upper"):
+                        upper_bound = srs[name] + self.mechanism.bounds(pct)[1]
+                        srs[name] = upper_bound
+                    else:
+                        counts = self.mechanism.release(srs[name])
+                        counts[counts < 0] = 0
+                        srs[name] = counts
+                        srs = srs.filter(name, ">", self.tau)
```

Comment:
This line is probably unnecessary, but defer fix to issue #24

Reply:
I see. Sure, keeping it as-is then.